nb2workflow/deploy.py

```diff
@@ -298,15 +308,40 @@ def _build_with_kaniko(git_origin,
         return container_metadata
 
 
-def _build_with_docker(git_origin, 
+def _extract_resource_requirements(local_repo_path, ontology_URL=default_ontology_URL):
+    sp.check_call([
+        "wget", ontology_URL, "-O", "ontology.ttl"
+    ])
+    ontology = Ontology("ontology.ttl")
```

Comment:
I confirm, that's possible, no need to write a file. Missed this point while reviewing

Reply:
@dsavchenko in fact passing URL as a parameter to Ontology didn't always work in my tests when I was using different URLs e.g. "https://raw.githubusercontent.com/oda-hub/ontology/storage_resource_annotations/ontology.ttl". In this case I get parser [exception](https://github.com/oda-hub/nb2workflow/files/14586629/exception.txt)

Reply:
That's because the content-type is set by github to be `text-plain` (on purpose, due to security implications, I believe) and this is used by rdflib to guess the ontology format, and it can't obviously be used. 
From https://odahub.io/ontology/ontology.ttl the ontology is served with the correct content type `text/turtle`, so there is no such problem. 
If you need to use the ontology from the specific commit, you can download it in the github workflow script and set the `--ontology-url` to be that local file.

Reply:
BTW, because `ontology-url` could also be just local path, it's better to change the name of this argument

Reply:
Done